imhamt/src/lib.rs

```diff
@@ -281,8 +250,8 @@ mod tests {
         true
     }
 
-    #[quickcheck]
-    fn insert_equivalent(xs: Vec<(String, u32)>) -> bool {
+    #[proptest]
+    fn insert_equivalent(xs: Vec<(String, u32)>) {
```

Comment:
This is one of these places where we may not need the `String` generator; could do with a simpler key type, so that case reduction does not get lost trying to simplify the string as a Unicode christmas tree.